src/page-modules/assistant/server/journey-planner/index.ts

```diff
@@ -122,7 +124,43 @@ export function createJourneyApi(
         result.data.trip,
       );
 
-      const validated = tripSchema.safeParse(data);
+      const trip: RecursivePartial<TripData> = data;
+
+      let tripPatterns = data.tripPatterns || [];
+      let searchAttempt = 1;
+
+      const MINIMUM_NUMBER_OF_TRIP_PATTERNS = 5;
```

Comment:
Is this the same behaviour as in the app, with the retries and minimum number of patterns to fetch?

Reply:
I see that the app searches for 8 initial hits with a maximum of 5 chained searches. I can update to use the same config as the app 👍